src/View/Admin/Article/index.html.twig

```diff
@@ -19,24 +19,29 @@
             {{ articles.error ?? '' }}
 
             {% for article in articles %}
+                {% set status =
```

Comment:
you could add a list like {title : brouillon, class: draft} to optimize those lines